modules/core/02-client/keeper/grpc_query_test.go

```diff
@@ -119,26 +119,11 @@ func (suite *KeeperTestSuite) TestQueryClientStates() {
 		{
 			"empty pagination",
 			func() {
-				localhost := types.NewIdentifiedClientState(exported.LocalhostClientID, suite.chainA.GetClientState(exported.LocalhostClientID))
-				expClientStates = types.IdentifiedClientStates{localhost}
+				expClientStates = nil
 				req = &types.QueryClientStatesRequest{}
 			},
 			true,
 		},
-		{
-			"success, only localhost",
```

Comment:
grpc should no longer return localhost in its query. There's no information to obtain, as it would just return back the height of the request

Reply:
that makes sense!